apiserver/modelmanager/modelmanager.go

```diff
@@ -593,32 +573,74 @@ func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag) (params.ModelInfo, er
 		info.CloudCredentialTag = cloudCredentialTag.String()
 	}
 
-	authorizedOwner := m.authCheck(owner) == nil
-	for _, user := range users {
-		if !authorizedOwner && m.authCheck(user.UserTag) != nil {
-			// The authenticated user is neither the owner
-			// nor administrator, nor the model user, so
-			// has no business knowing about the model user.
-			continue
-		}
+	// All users with access to the model can see the SLA information.
+	info.SLA = &params.ModelSLAInfo{
+		Level: model.SLALevel(),
+		Owner: model.SLAOwner(),
+	}
 
-		userInfo, err := common.ModelUserInfo(user, st)
-		if err != nil {
-			return params.ModelInfo{}, errors.Trace(err)
+	// If model is not alive - dying or dead - or if it is being imported,
+	// there is no guarantee that the rest of the call will succeed.
+	// For these models we can ignore NotFound errors coming from persistent layer.
+	// However, for Alive models, these errors are genuine and cannot be ignored.
+	ignoreNotFoundError := false
+	if model.Life() != state.Alive || model.MigrationMode() == state.MigrationModeImporting {
```

Comment:
the above lines can be collapsed to a single line
ignoreNotFoundErr := model.Life() != state.Alice || model.MigrationMode() == state.MigrationModeImporting